src/components/Section/index.tsx

```diff
@@ -98,53 +98,51 @@ function Section({
     const illustrationContainerStyle: StyleProp<ViewStyle> = StyleUtils.getBackgroundColorStyle(illustrationBackgroundColor ?? illustration?.backgroundColor ?? theme.appBG);
 
     return (
-        <>
-            <View style={[styles.pageWrapper, styles.cardSectionContainer, containerStyles, (isCentralPane || !!illustration) && styles.p0]}>
-                {cardLayout === CARD_LAYOUT.ICON_ON_TOP && (
-                    <IconSection
-                        icon={icon}
-                        iconContainerStyles={[iconContainerStyles, styles.alignSelfStart, styles.mb3]}
-                    />
-                )}
-                {!!illustration && (
-                    <View style={[styles.w100, styles.dFlex, styles.alignItemsCenter, styles.justifyContentCenter, illustrationContainerStyle]}>
-                        <View style={[styles.cardSectionIllustration, illustrationStyle]}>
-                            <Lottie
-                                source={illustration}
-                                style={styles.h100}
-                                webStyle={styles.h100}
-                                autoPlay
-                                loop
-                            />
-                        </View>
-                        {overlayContent?.()}
+        <View style={[styles.pageWrapper, styles.cardSectionContainer, containerStyles, (isCentralPane || !!illustration) && styles.p0]}>
+            {cardLayout === CARD_LAYOUT.ICON_ON_TOP && (
+                <IconSection
+                    icon={icon}
+                    iconContainerStyles={[iconContainerStyles, styles.alignSelfStart, styles.mb3]}
+                />
+            )}
+            {!!illustration && (
```

Comment:
I'd consider keeping one convention. E.g. [here](https://github.com/Expensify/App/pull/36849/files#diff-39be74c7a910e9671d67b25e28736c789ed518dbfbb2f6e0e57bbca058bb53fdR79) we are casting the value to bool with `Boolean()` constructor, and here we are doing it with `!!`

Reply:
I think the recent convention is to use `!!`, but I don't want to touch this in this PR. One day we will clean whole repo  👍